Data/ByteString/Internal.hs

```diff
@@ -153,19 +153,24 @@ import GHC.IOBase               (IO(IO),RawBuffer,unsafeDupablePerformIO)
 #endif
 
 import GHC.ForeignPtr           (ForeignPtr(ForeignPtr)
-                                ,newForeignPtr_, mallocPlainForeignPtrBytes)
+#if __GLASGOW_HASKELL__ < 900
+                                , newForeignPtr_
+#endif
+                                , mallocPlainForeignPtrBytes)
+
 #if MIN_VERSION_base(4,10,0)
 import GHC.ForeignPtr           (plusForeignPtr)
 #else
-import GHC.Types                (Int (..))
```

Comment:
[According to CI](https://travis-ci.org/github/haskell/bytestring/jobs/732487160), it is probably better to restore this line,

Reply:
Yeah, I'm doing the Travis-guided search.

Reply:
Should be fine now.